libraries/AP_HAL_Linux/RCInput_ZYNQ.cpp

```diff
@@ -45,13 +45,17 @@ void RCInput_ZYNQ::init()
 void RCInput_ZYNQ::_timer_tick()
 {
     uint32_t v;
-
```

Comment:
This is super picky, but I'd rather keep this empty line here for the sake of readability.